.pre-commit-config.yaml

```
@@ -0,0 +1,27 @@
+repos:
+  - repo: https://github.com/humitos/mirrors-autoflake
+    rev: v1.1
+    hooks:
+      - id: autoflake
+        args: ['-i', '--remove-all-unused-imports']
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args: ['-l', '88']
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        name: isort (python)
+        args: ['-sl', '--profile', 'black']
+      - id: isort
+        name: isort (pyi)
+        types: [ pyi ]
+        args: ['-sl', '--profile', 'black']
+
+  - repo: https://github.com/pycqa/flake8
```

```suggestion
  - repo: https://github.com/pycqa/flake8
```

> there are no empty lines by convention

In that case, this empty line can be deleted. Having said that, I think line breaks in the yaml file make different sections easier to identify.

---

let's go for empty lines. readability over rules